src/components/Breadcrumb/Breadcrumb.test.jsx

```diff
@@ -50,39 +50,4 @@ describe('Breadcrumb with overflow', () => {
     );
     expect(container.querySelector('.breadcrumb--overflow')).toBeTruthy();
   });
-
-  describe('has dev console warning(s)', () => {
-    const originalDev = window.__DEV__;
-    const originalResizeObserver = window.ResizeObserver;
-
-    beforeAll(() => {
-      jest.spyOn(console, 'error').mockImplementation(() => {});
-    });
-
-    beforeEach(() => {
-      window.__DEV__ = true;
-      window.ResizeObserver = undefined;
-    });
-
-    afterEach(() => {
-      console.error.mockClear();
-      window.__DEV__ = originalDev;
-      window.ResizeObserver = originalResizeObserver;
-    });
-
-    afterAll(() => {
-      console.error.mockRestore();
-    });
-
-    it('when ResizeObserver is not supported in the current environment', () => {
-      render(
-        <Breadcrumb {...commonProps} hasOverflow>
-          <BreadcrumbItem href="#">Breadcrumb 1</BreadcrumbItem>
-          <BreadcrumbItem href="#">Breadcrumb 2</BreadcrumbItem>
-          <BreadcrumbItem href="#">Breadcrumb 3</BreadcrumbItem>
-        </Breadcrumb>
-      );
-      expect(console.error).toHaveBeenCalledTimes(1);
-    });
-  });
```

Comment:
Why is this being removed? It ensures we only pop the console warning under the proper circumstances. This test was written when we had a bug raised about the warning condition being wrong.